browser/lib/search.js

```diff
@@ -0,0 +1,43 @@
+import _ from 'lodash'
+
+export default function searchFromNotes(data, search) {
+  let notes = data.noteMap.map((note) => note)
+  if (search.trim().length === 0) return []
+  let searchBlocks = search.split(' ')
+  searchBlocks.forEach((block) => {
+    if (block.match(/^#.+/)) {
+      const tag = block.match(/#(.+)/)[1]
+      notes = findByTag(notes, tag)
+    } else {
+      notes = findByWord(notes, block)
+    }
+  })
+  return notes
+}
```

Comment:
Thank you for changing them into the module :smile:
And probably, I made a mistake :fearful:. I assume arranged interface is better (regarding `findByTag()` and `findByWord`). So, you should change `findByTag()`. It's better that to cut out the tag from a block inside the method.

```
searchBlocks.forEach((block) => {
  if (block.match(/^#.+/)) {
     notes = findByTag(notes, block)
  } else {
     notes = findByWord(notes, block)
  }
})
```

And also, you can make a method such as `findByTagOrWord(block)` to combine the methods for finding. (I'm not sure the argument name (block) is better or not :face_with_thermometer:)